instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy

```diff
@@ -144,7 +145,7 @@ class TwilioClientTest extends AgentTestRunner {
         }
         span(1) {
           name "MessageCreator.create"
-          kind CLIENT
+          kind INTERNAL
```

Comment:
Is this change ok?

Reply:
well... it seemed better than having nested `CLIENT` spans. do u think better to suppress the nested `apache-httpclient` spans?

Reply:
Ah yeah I think we'd normally suppress them.

Reply:
done